osgi-integration/src/test/java/com/ericsson/bss/cassandra/ecchronos/osgi/ITTestECChronos.java

```diff
@@ -35,19 +35,18 @@
 import com.ericsson.bss.cassandra.ecchronos.connection.JmxConnectionProvider;
 import com.ericsson.bss.cassandra.ecchronos.connection.NativeConnectionProvider;
 import com.ericsson.bss.cassandra.ecchronos.core.JmxProxyFactory;
-import org.apache.felix.scr.Component;
-import org.apache.felix.scr.ScrService;
 import org.ops4j.pax.exam.Configuration;
 import org.ops4j.pax.exam.Option;
 import org.ops4j.pax.exam.junit.PaxExam;
 import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
 import org.ops4j.pax.exam.spi.reactors.PerClass;
 import org.osgi.framework.BundleContext;
+import org.osgi.service.component.runtime.ServiceComponentRuntime;
+import org.osgi.service.component.runtime.dto.ComponentDescriptionDTO;
 import org.osgi.util.tracker.ServiceTracker;
 
-import static junit.framework.TestCase.assertEquals;
 import static junit.framework.TestCase.assertNotNull;
-import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.*;
```

Comment:
The assertj framework is used on most other places. Perhaps better to stay consistent with that?